### PR TITLE
fix: special characters on localized Windows

### PR DESCRIPTION
**Pull request by zardoy**
For now, this module doesn't work on my windows machine (all tests are failing) because stdout look like that: `Size:         66�968�762 bytes`

This pr enables csv format which doesn't generate these weird characters at all.

Current du.exe usage (just in case):

```console
usage: du.exe [-c[t]] [-l <levels> | -n | -v] [-u] [-q] <directory>
   -c     Print output as CSV. Use -ct for tab delimiting.
          Use -nobanner to suppress banner.
   -l     Specify subdirectory depth of information (default is one level).
   -n     Do not recurse.
   -q     Quiet.
   -nobanner
          Do not display the startup banner and copyright message.
   -u     Count each instance of a hardlinked file.
   -v     Show size (in KB) of all subdirectories.

CSV output is formatted as:
Path,CurrentFileCount,CurrentFileSize,FileCount,DirectoryCount,DirectorySize,DirectorySizeOnDisk
```

### CODE REVIEW

**Comment by simoneb** [comment]
continuous integration is running tests on windows and they pass. do you know what's the difference between the CI environment and your local environment?

**Comment by zardoy** [comment]
https://github.com/sebhildebrandt/systeminformation/issues/317#issue-538964558 This can explain everything

**Comment by zardoy** [comment]
But it doesn't matter. CSV format returns only numbers. This it stdout output on my machine:
```console
Path,CurrentFileCount,CurrentFileSize,FileCount,DirectoryCount,DirectorySize,DirectorySizeOnDisk
 
"C:\Users\Professional\Desktop\repos\local-repo-stats\node_modules",1,8100,2535,205,66968765,85561344
```

**Comment by zardoy** [comment]
Sorry, I have accidently pushed another commit. Now everything should be okay.

File: index.js [hunk 1]
```diff
@@ -8,15 +8,15 @@ function fastFolderSize(target, cb) {
...
-      `"${path.join(__dirname, 'bin', 'du.exe')}" -nobanner -accepteula .`,
+      `"${path.join(__dirname, 'bin', 'du.exe')}" -nobanner -accepteula -q -c .`,
       { cwd: target },
       (err, stdout) => {
         if (err) return cb(err)
 
-        const match = /Size:\s+(.+) bytes/.exec(stdout)
-        const bytes = Number(match[1].replace(/\.|,/g, ''))
+        // query stats indexes from the end since path can contain commas as well
+        const stats = stdout.split("\n")[1].split(",")
```
Comment (simoneb) [inline index.js:17]: can you use single quotes for consistency?

**Review by simoneb — APPROVED** [review]
LGTM, one comment about quote usage and good to go. Thanks!

File: index.js (hunk 1 above)
Comment (zardoy) [reply index.js]: Sure! But I think it would be good to add a eslint rule for that `eslint quotes: ["error", "single"]`

Comment (simoneb) [reply index.js]: Yeah I thought it was there. There's prettier but not integrated with eslint, I'll put that in place at some point

**Pull request merged**